### PR TITLE
Reworked exome-seq tutorial

### PR DESCRIPTION
**Pull request by wm75**
Promised to do this long ago, finally done.

Basically, everything has been rewritten since the new version uses new (I think more interesting) input data (thanks @gmauro).

### CODE REVIEW

File: topics/variant-analysis/tutorials/exome-seq/tutorial.md [hunk 1]
```diff
@@ -25,290 +34,902 @@ contributors:
...
-> > - [Identifying autosomal dominant variants underlying Mendelian disease](https://s3.amazonaws.com/gemini-tutorials/Gemini-Dominant-Tutorial.pdf)
+> > - [Introduction to GEMINI](https://s3.amazonaws.com/gemini-tutorials/Intro-To-Gemini.pdf)
+> > - [Identifying *de novo* mutations with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-DeNovo-Tutorial.pdf)
+> > - [Identifying recessive gene candidates with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-Recessive-Tutorial.pdf)
+> > - [Identifying dominant gene candidates with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-Dominant-Tutorial.pdf)
 > {: .details}
-> And for a more detailed tutorial on variant data generation in Galaxy, have a
-> look at the [tutorial on diploid variant calling](https://github.com/nekrut/galaxy/wiki/Diploid-variant-calling).
-{: .comment}
+
```
Comment (bebatut) [inline topics/variant-analysis/tutorials/exome-seq/tutorial.md]: Is there any interesting results in the output? Could you add a question box or something to inspect and think about the results and relate to the original question? Thanks

File: topics/variant-analysis/tutorials/exome-seq/tutorial.md [hunk 2]
```diff
@@ -25,290 +34,902 @@ contributors:
...
+>
+> 7. Rename the datasets
+>
+>    For datasets that you upload via a link, Galaxy will pick the link
+>    address as the dataset name, which you will likely want to shorten to
+>    just the file names.
+>
+>    {% include snippets/rename_dataset.md %}
+>
+> 8. Add appropriate tags to the datasets
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/variant-analysis/tutorials/exome-seq/tutorial.md]: yeah, that is somewhat annoying, but it's the price we pay :smile: for reusing the input data from https://gmauro.github.io/galaxy-training/exome_analysis.html, which has chosen this name. It's way more common practice to talk of father/mother/child in trio analyses, but I can't change the SM field in the BAM input file.

File: topics/variant-analysis/tutorials/exome-seq/tutorial.md [hunk 3]
```diff
@@ -25,290 +34,902 @@ contributors:
...
+>     > {% icon tool%} tool to see if this is the case.
+>     {: .comment}
 >
->    > ### {% icon comment %} Comments
->    > You should now have 3 annotated variant files, from mother, father and the
->    > patient. It might be a good idea to rename them accordingly.
->    {: .comment}
+>     Use **SnpEff Download** {% icon tool %} to download genome annotation
+>     database `hg19`.
+> 2. Create a PED-formatted pedigree dataset like this
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/variant-analysis/tutorials/exome-seq/tutorial.md]: To make the pedigree dataset usable with GEMINI load all that's needed is to set its datatype to `tabular`. A fact I had forgotten to mention though :( Will fix it and thanks for catching!

I'm going to emphasize the significance of the sample names a lot more in the revised version, thanks.

However, I think it is slightly better to leave the dataset generation step in its current place. This way I imagine it's a bit easier for an instructor to get feedback from training participants whether they all have their required datasets ready. No need to interrupt everybody's work again shortly after.

File: topics/variant-analysis/tutorials/exome-seq/tutorial.md [hunk 4]
```diff
@@ -25,290 +34,902 @@ contributors:
...
 >
-> 3. Combine all 3 files into one with the tool **VCFcombine** from
-> **VCF Tools** {% icon tool %}
+>    ```
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/variant-analysis/tutorials/exome-seq/tutorial.md]: I'll add the optional header line of the file, which kind of makes the whole thing self-explanatory.

File: topics/variant-analysis/tutorials/exome-seq/tutorial.md [hunk 5]
```diff
@@ -25,290 +34,902 @@ contributors:
...
+>
+>    {% include snippets/import_from_data_library.md %}
+>
+> 3. Check that the newly created datasets in your history have their
+>    datatypes assigned correctly, and fix any missing or wrong
+>    datatype assignment
+>
+>    {% include snippets/change_datatype.md datatype="fastqsanger" %}
+>
+>    {% icon trophy %} Congratulations for obtaining the datasets required for
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/variant-analysis/tutorials/exome-seq/tutorial.md]: Hmm, I'm not 100% happy with the current form either, but I've spent *a lot* more time than I wanted on exactly this question already.
The problem with moving the premapped datasets steps to the postprocessing section is that you'd have to duplicate the final common steps 7-10.
I'd say the current version is good enough to release it into the wild. Let me gain a first round of training experience with it, then I'll know better whether it causes problems for participants, or for me explaining things, and I can update it accordingly.

File: topics/variant-analysis/tutorials/exome-seq/tools.yaml [hunk 6]
```diff
@@ -1,41 +1,67 @@
 ---
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/variant-analysis/tutorials/exome-seq/tools.yaml]: I've removed it.

**Comment by wm75** [comment]
@bebatut I've addressed pretty much all your suggestions (exception: see the one remaining unresolved conversation). Was quite a bit of work, but it really helped to improve the tutorial. Thanks!

**Comment by wm75** [comment]
Thanks @bebatut. These were reasonable changes :)

**Comment by bgruening** [comment]
grafik

**Pull request merged**